csharp/src/Ice/IceSSL/SSLEngine.cs

```diff
@@ -120,22 +123,14 @@ internal void initialize()
             {
                 throw new Ice.InitializationException($"IceSSL: CA certificate file not found: {certAuthFile}");
             }
-
             try
             {
-                // Try using PEM format
-                try
+                if (Path.GetExtension(certAuthFile).Equals(".pem", StringComparison.OrdinalIgnoreCase))
```

Comment:
According to this random website (https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/#ftoc-heading-2) PEM files can have several different extensions. Maybe this isn't such a good and we just should try to parse all the time?